react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+import './ToggleSwitch.css'
+
+export default function ToggleSwitch() {
+  let [clicked, setClicked] = useState(false);
+
+  function handleClick() {
+    setClicked(!clicked)
+  }
+  return (
+    <div className="div-box">
+      <lable className="switch">
+        <input type={clicked ? "checkbox" : "unchecked"} checked></input>
```

Comment:
I think this is not what you intended? This will set the input type to `unchecked` if clicked is false. I think you meant to replaced `checked` with `unchecked`. The easiest way to do that is to realize that `checked` is a boolean prop and use it like this:
`checked={!clicked}`